### PR TITLE
Visitor can review his order

### PR DESCRIPTION
**Pull request by mantanquzzmo**
[PT](https://www.pivotaltracker.com/story/show/170152675)

- adds page for user to see added items
- adds link in navbar

Skärmavbild 2019-12-14 kl  16 45 59

### CODE REVIEW

File: features/step_definitions/background_step.rb [hunk 1]
```diff
@@ -13,3 +13,12 @@
     create(:user, user_attributes)
   end
 end
+
+Given("the following product has been added to order") do |table|
+  table.hashes.each do |order_attributes|
+  order = @user.orders.create
```
Comment (sundqvistjohan) [inline features/step_definitions/background_step.rb]: Indentation, line 19-21

**Comment by sundqvistjohan** [comment]
The last commit removes the controller and view. What is going on there? There is a model Order with a relation (via acts_as_shopping_cart) to OrderItems, propose to create a OrderController instead of OrderedItemsController

**Review by sundqvistjohan — CHANGES_REQUESTED** [review]

File: features/step_definitions/background_step.rb (hunk 1 above)
Comment (mantanquzzmo) [reply features/step_definitions/background_step.rb]: yes

**Comment by mantanquzzmo** [comment]
> The last commit removes the controller and view. What is going on there? There is a model Order with a relation (via acts_as_shopping_cart) to OrderItems, propose to create a OrderController instead of OrderedItemsController

agree and sorted

**Pull request closed without merging**